std/didexchange/invitation/translate.go

```diff
@@ -20,24 +21,60 @@ func decodeB64(str string) ([]byte, error) {
 	return data, err
 }
 
+func convertFromOOB(invBytes []byte) (i Invitation, err error) {
+	defer err2.Returnf(&err, "oob conversion")
+
+	var oobInv OOBInvitation
+	try.To(json.Unmarshal(invBytes, &oobInv))
+
+	assert.D.True(len(oobInv.Services) > 0)
```

Comment:
FYI, `err2` has the `assert` package, which uses generics. Unfortunately, we cannot use generics with the methods. Go doesn't support that. So, the `D` and `P` have to be dealt with. The `assert` package uses `assert.DefaultAsserter` for that. It's not optimal because Go doesn't have thread local storage. But if the project which uses the `assert` package needs only one type of asserter most of the time, it's fine.

Anyway, thanks to generics there is functions like `assert.SNotEmpty(oobInv.Servics))`. S stands for slice. Others are C = chan, M = map.

All of this was just to share information. When I have integrated the first PR where `testify` is replaced with `err2/assert`, I'll explain more.

Reply:
Great! Let's also fix these later, we'll have to revisit this anyway.